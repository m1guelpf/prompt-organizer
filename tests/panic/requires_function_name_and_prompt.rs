use prompt_organizer::prompt;

prompt! {requires_prompt}
prompt! {"This is only the prompt, no function name is provided"}
prompt! {}
prompt!();

fn main() {}
