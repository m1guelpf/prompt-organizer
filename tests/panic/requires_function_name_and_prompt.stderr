error: expected `,`
 --> tests/panic/requires_function_name_and_prompt.rs:3:1
  |
3 | prompt! {requires_prompt}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `prompt` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected identifier
 --> tests/panic/requires_function_name_and_prompt.rs:4:10
  |
4 | prompt! {"This is only the prompt, no function name is provided"}
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: unexpected end of input, expected identifier
 --> tests/panic/requires_function_name_and_prompt.rs:5:1
  |
5 | prompt! {}
  | ^^^^^^^^^^
  |
  = note: this error originates in the macro `prompt` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected end of input, expected identifier
 --> tests/panic/requires_function_name_and_prompt.rs:6:1
  |
6 | prompt!();
  | ^^^^^^^^^
  |
  = note: this error originates in the macro `prompt` (in Nightly builds, run with -Z macro-backtrace for more info)
